redash/models.py

```diff
@@ -493,8 +493,7 @@ def to_dict(self, all=False, with_permissions_for=None):
                 DataSourceGroup.group == with_permissions_for,
                 DataSourceGroup.data_source == self).one()[0]
 
-        doc_url = self.options.get('doc_url', schema['properties'].get(
-            'doc_url', {}).get('default'))
+        doc_url = self.options.get('doc_url')
```

Comment:
The problem is it breaks showing values to the users for a data source pretty bad. I had to go back to commits from November to find this bug. I tried shifting the `schema` definition around as Allan had done before, but it did not resolve the issue.